miso-web/src/main/webapp/scripts/box_ajax.js

```diff
@@ -468,7 +479,11 @@ Box.ui = {
   },
   
   discardOneItem: function() {
-    var selectedPosition = Box.utils.getPositionString(Box.visual.selected.row, Box.visual.selected.col);
+    if (Box.visual.selectedItems.length !== 1) {
+      alert('Select a single tube to discard');
```

Comment:
Leaving the alerts for consistency within the existing page. These three cases are also double-checks and should never be seen because the controls should be disabled by the same condition that causes these alerts